packages/eslint-plugin-pf-codemods/test/rules/v5/menu-moved-ariaLabel.js

```diff
@@ -0,0 +1,84 @@
+const ruleTester = require("../../ruletester");
+const rule = require("../../../lib/rules/v5/menu-moved-ariaLabel");
+
+ruleTester.run("menu-moved-ariaLabel", rule, {
+  valid: [
+    {
+      code: `import { Menu, MenuList } from '@patternfly/react-core'; <Menu><MenuList aria-label="tester" /></Menu>`,
+    },
+    {
+      code: `import { Menu } from '@patternfly/react-core'; <Menu />`,
+    },
+    {
+      // No @patternfly/react-core import
+      code: `<Menu aria-label="tester"><MenuList /></Menu>`,
+    },
+    {
+      // No @patternfly/react-core import
+      code: `<Menu aria-label="tester" />`,
+    },
+  ],
+  invalid: [
+    {
+      code: `import { Menu } from '@patternfly/react-core'; <Menu aria-label="tester" />`,
+      output: `import { Menu } from '@patternfly/react-core'; <Menu  />`,
+      errors: [
+        {
+          message: `The aria-label prop has been removed from Menu and should be passed into MenuList instead. If using MenuGroup with the "label" prop, an aria-label on MenuList is not necessary.`,
+          type: "JSXElement",
+        },
+      ],
+    },
+    {
+      code: `import { Menu, MenuList } from '@patternfly/react-core'; <Menu aria-label="tester"><MenuList /></Menu>`,
+      output: `import { Menu, MenuList } from '@patternfly/react-core'; <Menu ><MenuList  aria-label="tester"/></Menu>`,
```

Comment:
with the rule change suggestion..

```suggestion
      output: `import { Menu, MenuList } from '@patternfly/react-core'; <Menu ><MenuList aria-label="tester"/></Menu>`,
```

wonder if we should also add a non-self closing MenuList as you addressed that in the rules